code/package.json

```diff
@@ -165,6 +173,7 @@
     "esbuild-loader": "^4.3.0",
     "eslint": "8.57.1",
     "eslint-import-resolver-typescript": "^3.6.1",
+    "eslint-plugin-compat": "^6.0.2",
```

Comment:
What does this do?

Reply:
It ensures that we're only using APIs that are compatible with the browsers defined in the browserslist configuration in code/package.json.

I'm thinking it's an experiment to see if it's useful or not.